<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessageReader</class>
 <widget class="QWidget" name="MessageReader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: white</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="1" column="0">
    <widget class="QFrame" name="buttonFrame">
     <property name="styleSheet">
      <string notr="true">
#buttonFrame {
background-image: url(:icons/backgrounds/291.jpg);
border: none;
padding: 0px;
}

#buttonFrame QPushButton {
color: #333;
border: none;
border-radius: 0px;
padding: 0px;
background: none;
}
#buttonFrame QProgressBar {
color: #222;
border: 2px solid #555;
border-radius: 5px;
padding: 5px;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 #433);
min-width: 80px;
}

#buttonFrame QLCDNumber {
color: #222;
border: 2px solid #555;
border-radius: 5px;
padding: 5px;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 #433);
min-width: 80px;
}


</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="replyButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="iconResources.qrc">
          <normaloff>:/icons/icons/round_and_up_icon&amp;48.png</normaloff>:/icons/icons/round_and_up_icon&amp;48.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="closeButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="iconResources.qrc">
          <normaloff>:/icons/icons/delete_icon&amp;48.png</normaloff>:/icons/icons/delete_icon&amp;48.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QTextEdit" name="messageEdit">
     <property name="styleSheet">
      <string notr="true">border:none</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="iconResources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>MessageReader</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>297</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
